src/epcpm/cantoxlsx.py

```diff
@@ -33,7 +34,7 @@
     top=CELL_SIDE, left=CELL_SIDE, right=CELL_SIDE, bottom=CELL_SIDE
 )
 CELL_FONT = openpyxl.styles.Font(size=8)
-
+NUMBERED_VARIANT_PATTERN = r"_(0[2-9]|1[0-9]|20)$"
```

Comment:
can you please explain (comment) why this regular exp is required ?  ... is it for a date, field with an specific range given an example

Reply:
The regex is required to target certain parameter names, such as FrequencyHighRideThrough_1_hertz_A (which has many numbered variations e.g. _01 to _20), so that they don't have a description section on each variation in EPC-CAN_for_manual.xlsx, but rather on the first one (the one ending in _01) only. This is to keep it consistent with how the manual was printed out previously.

Reply:
@dgarEPC FYI, Annie does a good job explaining which parameters this affects in the jira ticket.

Reply:
noted.  Let me check